move/axelar_gateway/sources/gateway.move

```diff
@@ -777,6 +786,8 @@ fun test_rotate_signers() {
         ctx,
     );
 
+    utils::multiple_events<events::SignersRotated>(2);
```

Comment:
this could be renamed to `assert_event_at` / `assert_event_at_index` maybe

Reply:
it is about having the right number of evens in the test so far, but i could change the logic to only returning the last event and then that name would make more sense